src/components/ConversionBar/ConversionBar.js

```diff
@@ -7,25 +8,39 @@ import {
   StyledCurrencyInput,
 } from "./styles";
 
-export default function ConversionBar(props) {
+function ConversionBar(props) {
   const [conversionAmount, setConversionAmount] = useState(1);
   const [coinAmount, setCoinAmount] = useState(
     ((1 / props.coinPrice) * 1).toFixed(6)
   );
 
   useEffect(() => {
     setConversionAmount(1);
-    setCoinAmount(((1 / props.coinPrice) * 1).toFixed(6));
+    setCoinAmount(
+      (1 / props.coinPrice) * 1 > 1
+        ? ((1 / props.coinPrice) * 1).toFixed(2)
+        : (1 / props.coinPrice).toFixed(5)
+    );
   }, [props.currency, props.coinPrice]);
 
   const handleCurrencyChange = (e) => {
     setConversionAmount(e.target.value);
-    setCoinAmount(((1 / props.coinPrice) * e.target.value).toFixed(6));
+    setCoinAmount(
+      (1 / props.coinPrice) * e.target.value > 0.1 ||
+        (1 / props.coinPrice) * e.target.value === 0
+        ? ((1 / props.coinPrice) * e.target.value).toFixed(2)
+        : ((1 / props.coinPrice) * e.target.value).toFixed(5)
+    );
```

Comment:
jesus, what's this? what are you trying to accomplish?

Reply:
its for the number of decimals places showing depending how big the number is. the best I could come up with at the time. wasnt working when I tried importing a formula from utils

Reply:
but if thats the only thing you are questioning im ok with it :)